### DISCUSSION TITLE
PWA vs Cordova application

### DISCUSSION THREAD

**Discussion by leostereo** [quasarframework/quasar#8851] [CLI - Capacitor / Cordova modes]
Hi guys , just want to ask for some clear concept.
Main pro using PWA over Cordova application is that you dont need to publish the application in the store since it can be installed directly when browsing (similar to quasar documentation page).
So ... what about Cordova over PWA ? Is it faster ... better...? ...What are pros uwing Cordoba over PWA ? 
Can PWA access all Cordoba application resourses ? (media , camera , geolocation, battery , networking , and so ?).
If so ... I dont see any reason to use Cordova.
Please , any correction u other consideration , would be wellcome.
Leandro.

**Comment by pikil** [comment]
One of the main pros Cordova has got is that it registers such things as badges and push-notifications, these are important when your users need to receive notifications to the app even if it is closed.

There are more, like Proximity, Switching between speakers, etc. (you would need to make a research on what you need to use in your app).

Another thing to keep in mind is that in PWA permissions are granted/denied to the browser (hence your PWA will depend on that when your user disables video on any other website), and in Cordova you get the permissions for your app instead independently.

Apart of that the difference is really minuscule as Cordova is a browser itself in general with access to the OS underlying features.
